model/project_aliases_test.go

```diff
@@ -493,6 +518,16 @@ func (s *ProjectAliasSuite) TestFindAliasInProjectOrRepo() {
 	found, err = FindAliasInProjectRepoOrConfig(pRef1.Id, "alias-5")
 	s.NoError(err)
 	s.Len(found, 0)
+
+	// Test project config
+	found, err = FindAliasInProjectRepoOrConfig(pRef1.Id, "alias-6")
+	s.NoError(err)
+	s.Len(found, 1)
```

Comment:
Is it worth checking that `found[0]` is the one you expect? Same with the otherone.

Reply:
Sure, added

Reply:
Need to `s.Require().Len` to avoid panicking in case found is empty.